src/Features/Onboarding.php

```diff
@@ -444,19 +530,20 @@ public static function update_help_tab() {
 	public static function reset_profiler() {
 		if (
 			! Loader::is_admin_page() ||
-			! isset( $_GET['reset_profiler'] ) || // WPCS: CSRF ok.
-			1 !== absint( $_GET['reset_profiler'] ) // WPCS: CSRF ok.
+			! isset( $_GET['reset_profiler'] ) // WPCS: CSRF ok.
 		) {
 			return;
 		}
 
+		$new_value = 1 === absint( $_GET['reset_profiler'] ) ? false : true;
+
 		$request = new \WP_REST_Request( 'POST', '/wc-admin/v1/onboarding/profile' );
 		$request->set_headers( array( 'content-type' => 'application/json' ) );
 		$request->set_body(
 			wp_json_encode(
 				array(
-					'completed' => false,
-					'skipped'   => false,
+					'completed' => $new_value,
```

Comment:
Since this is accessible to store owners, this would update both `skipped` and `completed` to true, right?

I'm wondering if this may create discrepancies in data later via tracks.  Maybe this should always set `completed` to false and `skipped` is the toggleable value?

Reply:
Yeah agreed. I've set completed to false, meaning they have to go back through the wizard to have it set to true, or they can 'skip' again.

https://github.com/woocommerce/woocommerce-admin/pull/2863/commits/52ef43e6054181fd540f56f12d5f12d62a2cc2ba